HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1166,33 +1163,34 @@ def self.apply_local_ventilation(model, vent_object, obj_name)
         end
         remaining_hrs -= 1
       end
+      obj_sch = HourlyByMonthSchedule.new(model, "#{obj_name} schedule", [daily_sch] * 12, [daily_sch] * 12, false, true, Constants.ScheduleTypeLimitsOnOff)
+      obj_sch_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Schedule Value')
+      obj_sch_sensor.setName("#{obj_name} sch s")
+      obj_sch_sensor.setKeyName(obj_sch.schedule.name.to_s)
+      obj_sch_sensors[vent_object.id] = obj_sch_sensor
+
+      equip_def = OpenStudio::Model::ElectricEquipmentDefinition.new(model)
+      equip_def.setName(obj_name)
+      equip = OpenStudio::Model::ElectricEquipment.new(equip_def)
+      equip.setName(obj_name)
+      equip.setSpace(@living_space)
+      if obj_type_name == Constants.ObjectNameMechanicalVentilationBathFan
+        equip_def.setDesignLevel(vent_object.fan_power * vent_object.quantity)
```

Comment:
Bath fan test ci diff is expected, the "quantity" is applied here as a bugfix.

Reply:
😮 Good catch!

Reply:
How about allowing an optional quantity input for kitchen fans too? That would simplify our code so we don't have to handle the bath fan as a special case. 99.9% of the time it will be 1, but that's okay.

Maybe for completeness, we should allow optional quantity inputs for all vent fan types?